doc/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
+#
+
+# You can set these variables from the command line.
+SPHINXOPTS    =
+SPHINXBUILD   = sphinx-build
+SOURCEDIR     = .
+BUILDDIR      = _build
+
+# Put it first so that "make" without argument is like "make help".
+help:
+	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+
+.PHONY: help Makefile
+
+# Catch-all target: route all unknown targets to Sphinx using the new
+# "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
+%: Makefile
+	notedown galgebra.md > galgebra_guide.ipynb
```

Comment:
I convert the .tex to a .md, fix it manually(a lot of work), and convert the .md to .ipynb using notedown and then use nbsphinx to convert the .ipynb to build the sphinx which is based on .rst.

My major concern for this method is that we have to maintain .md and generate  .ipynb manually and commit it to git (readthedocs doesn't run this makefile). And it deviate from the original tex due to some MathJax limitations(especially of equation referencing).

You may wonder, why go through all these trouble? Because none of the direct method works:

- The original doc in tex form was written in a not so rigorous way, most of the equations are not surrounded in `$` or `$$`, so pandoc didn't convert it properly to a .rst with math block.
- .md, even with the help of recommonmark, is not properly handled by sphinx, especially the formulas, but after notedown and nbsphinx, it's much easier to fix

Reply:
This sounds super annoying, the docs look amazing though

Reply:
Yeah, that's why I gave myself some time to step back and see if there's a better way. So far it's still the best working way.